build.gradle

```diff
@@ -655,7 +659,7 @@ task unpickIntermediaryJar(type: JavaExec, dependsOn: [mapIntermediaryJar, "cons
 	outputs.upToDateWhen { false }
 	group "unpick"
 
-	main "daomephsta.unpick.cli.Main"
+	mainClass.set "daomephsta.unpick.cli.Main"
```

Comment:
mainClass = "daomephsta.unpick.cli.Main"

Reply:
Hmm, I encountered errors when I used that regular format at first. Think the error must have been caused by something else, as gradle can report weird errors indeed (such as when I forgot to replace `finalizedBy` with `doLast` and gradle says "cannot convert a work result to a task")